Sources/MUXSDKStatsObjc/MUXSDKStats.m

```diff
@@ -20,206 +14,19 @@
 
 @implementation MUXSDKStats
 
-static MUXSDKDispatcher *_dispatcher;
-// Name => MUXSDKPlayerSoftware value.
-static NSMutableDictionary *_bindings;
-// Name => AVPlayerViewController or AVPlayerLayer or AVPlayer
-static NSMutableDictionary *_viewControllers;
-
-static MUXSDKPlayerBindingManager *_playerBindingManager;
-static MUXSDKCustomerPlayerDataStore *_customerPlayerDataStore;
-static MUXSDKCustomerVideoDataStore *_customerVideoDataStore;
-static MUXSDKCustomerViewDataStore *_customerViewDataStore;
-static MUXSDKCustomerCustomDataStore *_customerCustomDataStore;
-static MUXSDKCustomerViewerData *_customerViewerData;
-
-+ (void)initSDK {
-    if (!_bindings) {
-        _bindings = [[NSMutableDictionary alloc] init];
-    }
-    if (!_viewControllers) {
-        _viewControllers = [[NSMutableDictionary alloc] init];
-    }
-    if (!_customerPlayerDataStore) {
-        _customerPlayerDataStore = [[MUXSDKCustomerPlayerDataStore alloc] init];
-    }
-    if (!_customerVideoDataStore) {
-        _customerVideoDataStore = [[MUXSDKCustomerVideoDataStore alloc] init];
-    }
-    if (!_customerViewDataStore) {
-        _customerViewDataStore = [[MUXSDKCustomerViewDataStore alloc] init];
-    }
-    if (!_customerCustomDataStore) {
-        _customerCustomDataStore = [[MUXSDKCustomerCustomDataStore alloc] init];
-    }
-    if (!_playerBindingManager) {
-        _playerBindingManager = [[MUXSDKPlayerBindingManager alloc] init];
-        _playerBindingManager.customerPlayerDataStore = _customerPlayerDataStore;
-        _playerBindingManager.customerVideoDataStore = _customerVideoDataStore;
-        _playerBindingManager.customerViewDataStore = _customerViewDataStore;
-        _playerBindingManager.customerCustomDataStore = _customerCustomDataStore;
-        _playerBindingManager.viewControllers = _viewControllers;
-    }
-
-    // Provide EnvironmentData and ViewerData to Core.
-    MUXSDKEnvironmentData *environmentData = [self buildEnvironmentData];
-    MUXSDKViewerData *viewerData = [self buildViewerData];
-    MUXSDKDataEvent *dataEvent = [[MUXSDKDataEvent alloc] init];
-    [dataEvent setEnvironmentData:environmentData];
-    [dataEvent setViewerData:viewerData];
-    [MUXSDKCore dispatchGlobalDataEvent:dataEvent];
-}
-
-+ (MUXSDKEnvironmentData *)buildEnvironmentData {
-    MUXSDKEnvironmentData *environmentData = [[MUXSDKEnvironmentData alloc] init];
-    [environmentData setMuxViewerId:[self getUUIDString]];
-    /*
-    NSString *debugData = [MUXSDKConfigParser getNSStringForKey:@"debug" fromDictionary:config];
-    if (debugData) {
-        [environmentData setDebug:debugData];
-    }
-    */
-    return environmentData;
-}
-
-+ (MUXSDKViewerData *)buildViewerData {
```

Comment:
The logic in `buildViewerData` has been extracted out to a separate `MUXSDKPlaceholderViewerData` class.